docs/framework/configure-apps/file-schema/index.md

```diff
@@ -22,79 +22,78 @@ helpviewer_keywords:
+Elements used to create and use configuration sections for custom settings.
 
-[Trace and Debug Settings Schema](./trace-debug/index.md)
-Describes the elements that specify trace switches and listeners.
+[Trace and Debug Settings Schema](./trace-debug/index.md)\
+Elements that specify trace switches and listeners.
 
-[Compiler and Language Provider Settings Schema](./compiler/index.md)
-Describes the elements that specify compiler configuration for available language providers.
+[Compiler and Language Provider Settings Schema](./compiler/index.md)\
+Elements that specify compiler configuration for available language providers.
 
-[Application Settings Schema](application-settings-schema.md)
-Describes the elements that enable a Windows Forms or ASP.NET application to store and retrieve application-scoped and user-scoped settings.
+[Application Settings Schema](application-settings-schema.md)\
+Elements that enable a Windows Forms or ASP.NET application to store and retrieve application-scoped and user-scoped settings.
 
-[App Settings Schema](./appsettings/index.md)
+[App Settings Schema](./appsettings/index.md)\
 Contains custom application settings, such as file paths, XML Web service URLs, or any other custom configuration information for an application.
 
-[Web Settings Schema](./web/index.md)
-All elements in the Web settings schema, which includes elements for configuring how ASP.NET works with a host application such as IIS. Used in *Aspnet.config* files.
+[Web Settings Schema](./web/index.md)\
+Elements for configuring how ASP.NET works with a host application such as IIS. Used in *Aspnet.config* files.
 
-[Windows Forms Configuration Schema](winforms/index.md)
-All elements in the Windows Forms application configuration section, which includes customizations such as multi-monitor and high DPI support.
+[Windows Forms Configuration Schema](winforms/index.md)\
+All elements in the Windows Forms application configuration section, which includes customizations such as multi-monitor and high-DPI support.
 
-[WCF Configuration Schema](./wcf/index.md)
+[WCF Configuration Schema](./wcf/index.md)\
 All elements that enable you to configure WCF service and client applications.
 
-[WCF Directive Syntax](./wcf-directive/index.md)
+[WCF Directive Syntax](./wcf-directive/index.md)\
 Describes the `@ServiceHost` directive, which defines page-specific attributes used by the .svc compiler.
 
-[WIF Configuration Schema](windows-identity-foundation/index.md)
+[WIF Configuration Schema](windows-identity-foundation/index.md)\
 All elements of the Windows Identity Foundation (WIF) configuration schema.
 
 ## Related sections
 
-[Remoting Settings Schema](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/z415cf9a(v=vs.100))
+[Remoting Settings Schema](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/z415cf9a(v=vs.100))\
 Describes the elements that configure client and server applications that implement remoting.
```

Comment:
I think you missed this one, as well as some others below.

Reply:
These changes are correct. There is a blank line here and in the following elements

Reply:
@BillWagner, I meant the line that begins with `Describes the`, similar lines was updated and the `Describes the` was deleted.